docs/tutorials/emulation/02_dim_reduction.ipynb

```diff
@@ -149,24 +149,39 @@
    "source": [
     "from autoemulate.core.compare import AutoEmulate\n",
     "from autoemulate.transforms import *\n",
+    "from torch.distributions.transforms import ReshapeTransform\n",
     "import torch \n",
     "\n",
     "# Convert to tensors\n",
-    "x, y = torch.Tensor(X).float(), torch.Tensor(Y).float()\n",
+    "x, y = torch.Tensor(X).float(), torch.Tensor(Y).float().reshape(-1, n, n)\n",
     "\n",
     "# Split into train/test data\n",
     "test_idx = [np.array([13, 39, 30, 45, 17, 48, 26, 25, 32, 19])]\n",
     "train_idx = np.setdiff1d(np.arange(len(x)), test_idx)\n",
     "\n",
+    "# Run AutoEmulate\n",
     "ae = AutoEmulate(\n",
     "    x[train_idx], \n",
     "    y[train_idx],\n",
-    "    models=[\"GaussianProcess\"],\n",
+    "    models=[\"GaussianProcess\", \"GaussianProcessCorrelated\"],\n",
     "    x_transforms_list=[[StandardizeTransform()]],\n",
-    "    y_transforms_list=[[StandardizeTransform(), PCATransform(n_components=16), StandardizeTransform()]],\n",
-    "    n_splits=2,\n",
-    "    log_level=\"error\", \n",
-    ")"
+    "    y_transforms_list=[[\n",
+    "        ReshapeTransform((n, n), (n * n,)), # include torch transform to flatten first\n",
```

Comment:
I think it's really nice that we're using the ReshapeTransform from torch here, probably worth adding a note in the text above this cell to point out that torch transforms can be used.